javatests/arcs/android/common/resurrection/ResurrectionRequestTest.kt

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.common.resurrection
+
+import android.content.ComponentName
+import android.content.Intent
+import android.os.PersistableBundle
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import arcs.core.storage.StorageKey
+import arcs.core.storage.driver.RamDiskStorageKey
+import arcs.core.testutil.fail
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ResurrectionRequestTest {
+    @get:Rule
```

Comment:
FMI, what is `@get:Rule`? How is it different from `@Rule`?

Reply:
Without `get:`, you end up getting this:

```
org.junit.internal.runners.rules.ValidationError: The @Rule 'activityTestRule' must be public.
```

https://proandroiddev.com/fix-kotlin-and-new-activitytestrule-the-rule-must-be-public-f0c5c583a865

Seems to be due to the `val` being a property, and not a field. The alternative listed in that blog post is to do:

```kotlin
@Rule @JvmField val mything = MyRule()
```